TidepoolOnboardingKitUI/View Models/PreferredGlucoseUnitViewModel.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  PreferredGlucoseUnitViewModel.swift
+//  TidepoolOnboardingKitUI
+//
+//  Created by Darin Krauss on 1/29/21.
+//  Copyright © 2021 Tidepool Project. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKitUI
+
+class PreferredGlucoseUnitViewModel: ObservableObject {
```

Comment:
[nit] I suppose technically this is sort of a `ViewModel`, but it seems to me to simply be an `Observable`. I know we don't really have that naming pattern, but I have a slight preference to call this `PreferredGlucoseUnitObservable`.  Again, just slight.

Reply:
I'll update this name in the next set of work. :+1: